docs/gadget-devel/gadget-wasm-api-raw.md

```diff
@@ -280,6 +281,16 @@ Parameters:
 Return value:
 - (u32): Data handle on success, 0 on error
 
+#### `dataRelease(m uint32) uint32`
```

Comment:
IMO it should stay `dataRelease` since this releases the handle for the Data and not the dataarray. On the api side this is also a function on the `Data` typedef

Reply:
Since now we use a general `releaseHandle` function this discussion is not needed anymore